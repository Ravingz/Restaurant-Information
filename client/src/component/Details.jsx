import React from 'react';
class Details extends React.Component{
    constructor(props){
        super(props);
        this.state = {Clicked : false};
    }


    render(){
        return(
            <div className ="modal_content">
                <div onClick={this.props.updateStatus}>&times;</div>
            </div>
        )
    }
}
export default Details