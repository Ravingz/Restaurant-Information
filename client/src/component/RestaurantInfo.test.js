import React from 'react';
import { shallow } from 'enzyme';
import RestaurantInfo from './RestuarantInfo.jsx';
import Details from './Details.jsx'

describe('Components should render correctly', () => {
    test('RestaurantInfo should render', () => {
        const component = shallow(<RestaurantInfo />);

        expect(component).toMatchSnapshot();
    });
    test('Deails should render', () => {
        const component = shallow(<Details />);

        expect(component).toMatchSnapshot();
    });
    
});

// decribe('button component should respond', () => {
//     test('when details button is click, click event should be triggered', () =>{
//         const mockButtonClick = jest.fn();

//         const button= shallow()
//     })
// })



// describe('Test Button component', () => {
//     it('Test click event', () => {
//       const mockCallBack = jest.fn();
  
//       const button = shallow((<Button onClick={mockCallBack}>Ok!</Button>));
//       button.find('button').simulate('click');
//       expect(mockCallBack.mock.calls.length).toEqual(1);
//     });
//   });