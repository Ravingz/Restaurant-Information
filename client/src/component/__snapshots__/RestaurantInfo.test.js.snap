// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components should render correctly RestaurantInfo should render 1`] = `
<div>
  <div>
     
  </div>
  <div>
     
     Reviews 
    <Details />
  </div>
  <div>
     
  </div>
</div>
`;

exports[`Components should render correctly RestaurantInfo should render 2`] = `""`;
